src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -325,6 +325,15 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
     <comment>'{0}' is replaced with the resource that could not be downloaded.
 '{1}' is replaced with the integer (in milliseconds) of the timeout duration.</comment>
   </data>
+  <data name="Http_CommunicationFailedWithDetails" xml:space="preserve">
+    <value>Communication failed to the package source with "{1}" error.  Package source: '{0}'.</value>
+    <comment>{0} is the url of the package source.
+{1} is an error message from the networking stack.</comment>
+  </data>
+  <data name="Http_CommunicationFailed" xml:space="preserve">
+    <value>Communication failed to the package source.  Source: '{0}'.</value>
+    <comment>{0} is the url of the package source.</comment>
+  </data>
```

Comment:
Both of these strings appear to be unused.  Be sure to revert these changes in both files (.resx and .Designer.cs).

Reply:
ah yes. NU1305 existed for a bit... but back off due to fear of changing too much in other code paths. removed in next iteration.